packages/stable/src/__tests__/api/datamodels.int.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('Data models integration test', () => {
   const datamodelCreationDefinition: DataModelCreate = {
     externalId: `test_datamodel_${randomInt()}`,
     space: TEST_SPACE_NAME,
-    name: 'test_datamodel',
```

Comment:
Any reason for why this name had to be changed?

Reply:
I will revert it back. i think PR checks were failing and revert and see if it works